data/incipias/incipias.txt

```diff
@@ -0,0 +1,372 @@
+	names "incipias civilian"
+	cargo 50
+	personality
+		confusion 40
+		timid appeasing
+	variant 5
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbus (Civil)"
+	variant 4
+		"Nimbo Cirrus" 2
+	variant 3
+		"Gero"
+	variant 2
+		"Gero" 2
+	variant 2
+		"Gero"
+		"Nimbo Cirrus"
+	variant 1
+		"Nimbo Cirrus" 2
+		"Nimbus (Civil)"
+
+outfitter "Hicemus"
+	"Plasma Discharger"
+	"Stagnation Beam"
+	"Gas-Class Shield"
+	"Liquid-Class Shield"
+	"Metallic Hydrogen Cell"
+	"Cloud Piercer"
+
+	"MH Blue-Class Engine"
+	"MH Blue-Class Steering"
+	"MH Green-Class Engine"
+	"MH Green-Class Steering"
+	"MH Red-Class Engine"
+	"MH Red-Class Steering"
+
+shipyard "Hicemus"
+	"Nimbo Cirrus"
+	"Nimbo Stratus"
+	"Nimbus"
+	"Gero"
+
+news "Incipias common"
+	location
+		government "Conlatio" "Hicemus"
+	name
+		word
+			"Incipias Captain"
+			"Incipias Crew Member"
+	message
+		phrase
+			"incipias common hard defined"
+			"incipias common praise"
+
+phrase "incipias common"
+	phrase
+		"incipias common hard defined" 1
+		"incipias common praise" 5
+		"incipias common interest" 3
+		"incipias common advise" 3
+
+phrase "incipias common hard defined"
+	word
+		"You say you left this home-system? One day, I will also see other home-systems."
+		"The before-present was a more interesting time to fly a spaceship. Nowadays, a lot of people have one."
+		"I just got my crewmember-ship-license renewed. The government controls them strictly."
+		"There is nothing more exciting than to switch to turbine mode and fly through the landscape-clouds of our origin-home."
+		"It has been some time since the last great knowledge-breakthrough has been achieved in technological research-understanding. The government promises us that they are working on something, but my guess is that they have no clue what to do."
+		"I saw a plasma discharger with a leak-crack not so long ago. The ship looked good from the outside, but the inside was a firestorm."
+		"I have never seen something like your species. Why don't you float like us?"
+		"I was on a mining mission-job not so long ago when I saw a government ship drop a small box. The box accelerated very fast in one direction and disappeared with a flash."
+
+phrase "incipias common praise"
+	word
+		"These technologies"
+		"These ships"
+		"These death-weapons"
+		"These engines"
+		"These life-shields"
+	word
+		" "
+	word
+		"of yours"
+		"you have"
+		"you have installed"
+		"of your kind"
+	word
+		" "
+	word
+		"are very impressive."
+		"are very efficient."
+		"look very stylish."
+
+phrase "incipias common interest"
+	word
+		"What "
+	word
+		"have you seen"
+		"happened to you"
+		"did you encounter"
+		"wonders lie"
+	word
+		" "
+	word
+		"beyond our system-home"
+		"in the stars"
+		"away from safety"
+		"past our travel-borders"
+	word
+		"? "
+	word
+		"Were you impressed?"
+		"Was it fearsome?"
+		"Was it dangerous?"
+
+phrase "incipias common advise"
+	word
+		"See"
+		"Look at"
+		"Pay attention to"
+		"Light"
```

Comment:
```suggestion
		"Follow"
```
The current version doesn’t quite make sense.

Reply:
Well that's the thing though, when translating not everything will make sense, although if people find it very distrubing I will change it

Reply:
I’m not “people” by any means, so if you think it works better the way it is than keep it that way.

Reply:
I will just wait for more opinions

Reply:
Good.

Reply:
Wanderer dialogue is already full of things that don't quite translate well so this fits right in.

Reply:
If it won’t be hard to understand, it’s fine to keep it in. Seeing as how @roadrunner56 thought it worked, this can probably be resolved.